contracts/Authority.sol

```diff
@@ -26,24 +26,52 @@ contract Authority is DSRoles {
   uint8 adminRole = 1;
```

Comment:
I'm not sure if there's an issue at all here with `0` value roles. It's more of a suspicion

Reply:
There shouldn't be. The number used here determines which bit is used in the mask `DSRoles` uses to indicate that a user has a particular role. If we didn't use 0, we'd 'only' be able to have 255 roles rather than 256, and the LSB would never be used.